root/account/register.tt

```diff
@@ -18,6 +18,9 @@
                   For full details, please read our {doc|Privacy Policy}.',
                   {doc => { href => 'https://metabrainz.org/privacy' } }) -%]</p>
 
+        <p>[%- l('If you so choose, you will be able to remove any personal information from our services by
```

Comment:
"You may remove your personal information from our services anytime by deleting your account."

A little less stiff.